### ISSUE TITLE
Improve is_valid_in_api check for InitialWaterlevels

### ISSUE THREAD

**Issue by daanvaningen** [nens/threedi-modelchecker#196]
Currently save_to_api for initial waterlevel is able to run while initial waterlevel might not be uploaded: https://github.com/nens/threedi-api/issues/1672. This creates issues in making of simulation templates. Improve the is_valid_in_api function to make this more robust.

**Comment by Jonasvsl** [comment]
is this the correct repo for this ticket?

**Comment by daanvaningen** [comment]
Yes, the simulation template creation stuff is in the modelchecker
